ui/raidboss/data/03-hw/dungeon/sohm_al.ts

```diff
@@ -2,9 +2,17 @@ import Conditions from '../../../../../resources/conditions';
 import NetRegexes from '../../../../../resources/netregexes';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  playerMap?: { [name: string]: boolean };
+  something?: boolean;
```

Comment:
Seems like this `something` and the other one are unused?

Reply:
😇

Reply:
Thanks.  I'm going to blame VSCode for forever having unsaved changes after I commit.  <_<